x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/slm/SlmHealthIndicatorService.java

```diff
@@ -153,9 +153,10 @@ public HealthIndicatorResult calculate(boolean verbose, int maxAffectedResources
                             + policy.getName()
                             + "] had ["
                             + policy.getInvocationsSinceLastSuccess()
-                            + "] repeated failures without successful execution since ["
-                            + FORMATTER.formatMillis(policy.getLastSuccess().getSnapshotStartTimestamp())
-                            + "]"
+                            + "] repeated failures without successful execution"
+                            + (policy.getLastSuccess().getSnapshotStartTimestamp() != null
```

Comment:
Note that `getLastSuccess` can return `null` too. Let's check for that as well

Reply:
Good catch